### ISSUE TITLE
Clicking on the 'Add Record to Issue' Closes Main Menu

### ISSUE THREAD

**Issue by qdd8jtl** [CIMDBORG/CIMMigrationProject#33]
When clicking on the 'Add Record to Issue' on the  new hamburger menu, it opens up the Add New Record Form.  However, it also closes the main menu screen and only the Add New Record form is open.  The application has to be re-opened to navigate within the database again.

**Comment by brcox90** [comment]
Would this fix be acceptable?

- If the user hits submit or cancel on the screen, they'll be taken to the Issues DB menu screen. 

- If the user closes the window, the application will open the Database Menu screen

Personally, I think flow works best so that the user doesn't have a bunch of screens opening without others closing. The overall app should really strive to work one window at a time, with each window pathing either further into the application, or back to the main menu.

**Comment by qdd8jtl** [comment]
I agree!  I did notice that one window would close and the current would remain open which is great so you don't have 10 different windows open at the end of it all.  The only time that doesn't happen is if you click on the Error File database (double check me on that).  

I do like the idea of making sure the database main menu appears whether you hit Submit, Cancel, or the X on top right corner.  

Should we address the issue of having multiple windows open in a different issue?  I find that between Strategic Tasks, Browse Items, Weekly Review, and Manager tasks, I'm left with a ton of windows open at the end of the day.  Mostly due to getting interrupted in the middle of doing something and forgetting I had a window open already.

**Comment by NickD0w** [comment]
I like the idea, and agree with the overall flow thought, but takecare not to code it to close all windows. For instance I like to be able to have the browse items results open when adding a new issue to I can see the priority #'s.

**Comment by brcox90** [comment]
@qdd8jtl @NickD0w 
I think that should be a code clean up issue, separate from this. We would have to discuss it, to make sure we all agree on it. As Nick pointed out, there are some windows where it's beneficial to keep them open in certain cases, etc. This can be something for the team meeting. Simple addition to code that we can do as we update comments on each page.

**Comment by brcox90** [comment]
Also, this issue has been fixed.

**Comment by brcox90** [comment]
[Coding Changes.txt](https://github.com/CIMDBORG/CIMMigrationProject/files/4187234/Coding.Changes.txt)

@mikey2695 since things are being put on hold, I think we should push an update of what's ready to go right now since there are some item we needed implemented that were waiting on other things to get pushed. 

@NickD0w @qdd8jtl @tcoma-ups do you agree?

**Comment by NickD0w** [comment]
I would agree as long as we're confident the changes will not add more bugs.

**Comment by Pawel062** [comment]
Drop the changes in with a bottle of Raid. It'll keep the bugs out ;)

**Comment by brcox90** [comment]
@NickD0w pretty much all the changes available to get implemented are design changes, or Mike reloading fixes he made that got lost when we started moving code around and stuff.

**Comment by mikey2695** [comment]
I'm ready on my end, seems to be fine now.

**Comment by brcox90** [comment]
Fixed in last update
